developer_guides/topology/topology.rst

```diff
@@ -612,4 +651,4 @@ actually push both messages to a dot file:
 .. _here: https://www.alsa-project.org/main/index.php/ALSA_topology
 .. _SOFT: https://github.com/thesofproject/soft
 
-.. comment "master" has been replaced with "main"
\ No newline at end of file
+.. comment "master" has been replaced with "main"
```

Comment:
what is this modification about?

Reply:
don't know why this happen, looks it is introduced automatically when I edit this .rst file and not removable.

Reply:
@plbossart and @keyonjie That was a note I made to indicate that I had replaced references to "master" with "main." It was to make sure I used the correct change. I can remove it.